cmd/completion.go

```diff
@@ -23,8 +33,27 @@ default on Mac.  This can be installed by using homebrew:
 Once installed, bash_completion must be evaluated.  This can be done by adding the
 following line to the .bash_profile
 
-    $ source $(brew --prefix)/etc/bash_completion`,
-	Example:   `# cozy-stack completion bash > /etc/bash_completion.d/cozy-stack`,
+    $ source $(brew --prefix)/etc/bash_completion
+
+Zsh:
+
+  # If shell completion is not already enabled in your environment,
+  # you will need to enable it.  You can execute the following once:
+
+  $ echo "autoload -U compinit; compinit" >> ~/.zshrc
+
+  # To load completions for each session, execute once:
+  $ cozy-stack completion zsh > "${fpath[1]}/_cozy-stack"
+
+  # You will need to start a new shell for this setup to take effect.
+
+fish:
+
+  $ cozy-stack completion fish | source
+
+  # To load completions for each session, execute once:
+  $ cozy-stack completion fish > ~/.config/fish/completions/cozy-stack.fish
```

Comment:
Maybe instruct here to add completion for all users rather that only current user

Suggested change :
```patch
- $ cozy-stack completion fish > ~/.config/fish/completions/cozy-stack.fish
+ $ cozy-stack completion fish > /etc/fish/completions/cozy-stack.fish
```